metadata.display.yaml

```diff
@@ -66,7 +66,7 @@ spec:
         topic:
           name: topic
           title: Topic
-          regexValidation: ^(?!goog)[a-zA-Z][a-zA-Z0-9\-._~%+]{0,254}$
+          regexValidation: `^([a-fh-zA-FH-Z][a-zA-Z0-9\-._~%+]{0,254}|[gG]($|[^oO][a-zA-Z0-9\-._~%+]{0,253}|[oO]($|[^oO][a-zA-Z0-9\-._~%+]{0,252}|[oO]($|[^gG][a-zA-Z0-9\-._~%+]{0,251}))))$`
```

Comment:
The lint validation is failing with this regex. You can escape special characters to make is valid string in yaml